config/routes.rb

```diff
@@ -6,9 +6,6 @@
     mount Rswag::Api::Engine => '/api-docs'
   end
 
-  get "/docs/*id" => 'pages#show', as: :page, format: false
```

Comment:
maybe redirect this to https://github.com/ministryofjustice/hmpps-book-secure-move-api/wiki ?